### ISSUE TITLE
file.exists Error of internal_failure

### ISSUE THREAD

**Issue by paulshapiro** [GoogleCloudPlatform/gcloud-node#1248]
Hi there,

I'm doing a bunch of uploads which are preceded by a file.exists to prevent redundant uploads. It was working quite well through the night but I suddenly started to get an error of "internal_failure", with the stack trace below. I've tried deleting the files that it fails on, but same thing keeps happening on different files. This error is yielded by the file.exists call.

```
Error: internal_failure
    at Request._callback (/[path to…]-server-js/node_modules/google-auth-library/lib/transporters.js:78:15)
    at Request.self.callback ([path to…]-server-js/node_modules/google-auth-library/node_modules/request/request.js:198:22)
    at emitTwo (events.js:100:13)
    at Request.emit (events.js:185:7)
    at Request.<anonymous> ([path to…]-js/node_modules/google-auth-library/node_modules/request/request.js:1057:14)
    at emitOne (events.js:95:20)
    at Request.emit (events.js:182:7)
    at IncomingMessage.<anonymous> ([path to…]-js/node_modules/google-auth-library/node_modules/request/request.js:1003:12)
    at emitNone (events.js:85:20)
    at IncomingMessage.emit (events.js:179:7)
    at endReadableNT (_stream_readable.js:913:12)
    at _combinedTickCallback (node.js:383:13)
    at process._tickCallback (node.js:407:11)
```

–Paul

**Comment by paulshapiro** [comment]
I think this may be an issue in Google Cloud itself. Doing a gsutil command on the command line yields the same error. Going then to a `gcloud auth login` leads me to a page that 500s.

**Comment by paulshapiro** [comment]
Similarly, cannot deploy. I get

`ERROR: (gcloud.preview.app.deploy) internal_failure`

**Comment by stephenplusplus** [comment]
Same here, appears to be an issue with the oauth token server. @jgeewax have you heard of any downtime reports?

**Comment by stephenplusplus** [comment]
Update coming soon: https://status.cloud.google.com/

**Comment by paulshapiro** [comment]
@stephenplusplus working again, for me. Thanks!

**Comment by stephenplusplus** [comment]
Awesome! Here is the official summary:

The issue with Authentication Services should have been resolved for all affected projects as of 07:24 US/Pacific. We will conduct an internal investigation of this issue and make appropriate improvements to our systems to prevent or minimize future recurrence. We will provide a more detailed analysis of this incident once we have completed our internal investigation.
